easyconfigs/a/Arriba/Arriba-2.4.0-GCC-13.3.0.eb

```
@@ -0,0 +1,48 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'Arriba'
+version = '2.5.0'
+
+github_account = 'suhrig'
+homepage = 'https://github.com/%(github_account)s/%(namelower)s'
+description = """Arriba is a command-line tool for the detection of gene fusions from RNA-Seq data.
+ It was developed for the use in a clinical research setting. Therefore, short runtimes and high
+ sensitivity were important design criteria."""
+
+toolchain = {'name': 'GCC', 'version': '13.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/%(github_account)s/%(namelower)s/releases/download/v%(version)s']
+sources = ['%(namelower)s_v%(version)s.tar.gz']
+checksums = ['e7151037b0870dd952ed3c19765d93cd13b2cefd6374d81ff926e00c20caacfa']
+
+dependencies = [
+    ('bzip2', '1.0.8'),
+    ('HTSlib', '1.21'),
+    ('libdeflate', '1.20'),
+    ('STAR', '2.7.11b'),
+    ('zlib', '1.3.1'),
+    ('XZ', '5.4.5'),
+]
+
+# make target with dynamically linked libraries is called bioconda, default is a statically linked binary
+build_cmd_targets = 'bioconda'
+# otherwise it fails to find sam.h
+buildopts = 'CPATH=${EBROOTHTSLIB}/include/htslib:CPATH'
```

```suggestion
buildopts = 'CPATH=${EBROOTHTSLIB}/include/htslib:$CPATH'
```

Looks like this missing `$` has been lurking in the Arriba confs for a few versions!